### PR TITLE
View txns

### PR DESCRIPTION
**Pull request by alajko**
See Transactions

### CODE REVIEW

**Comment by alajko** [comment]
Screen Shot 2019-05-20 at 9 29 33 PM

**Comment by alajko** [comment]
On hover:

Screen Shot 2019-05-20 at 9 29 41 PM

File: puzzle/puzzle_vuejs/src/components/RedeemPanel.vue [hunk 1]
```diff
@@ -1,4 +1,12 @@
 <style scoped lang="less">
+a:link {
```
Comment (lzl124631x) [inline puzzle/puzzle_vuejs/src/components/RedeemPanel.vue]: no need for `:link`. Haven't seen such thing.

Comment (alajko) [reply puzzle/puzzle_vuejs/src/components/RedeemPanel.vue]: That is what is removing the underline from the link.

Comment (lzl124631x) [reply puzzle/puzzle_vuejs/src/components/RedeemPanel.vue]: I mean don't add `a:link`, just use `a`. `:link` is for un-visited link only https://developer.mozilla.org/en-US/docs/Web/CSS/:link

**Pull request closed without merging**